mysql-test/r/percona_utility_user.result

```diff
@@ -163,4 +163,9 @@ PROCESSLIST_ID	ATTR_NAME	ATTR_VALUE	ORDINAL_POSITION
 SELECT COUNT(DISTINCT PROCESSLIST_ID) FROM performance_schema.session_connect_attrs;
 COUNT(DISTINCT PROCESSLIST_ID)
 1
+SELECT COUNT(*) from performance_schema.threads where type='FOREGROUND';
+COUNT(*)
+3
```

Comment:
@dutow Is it expected that this number is increasing from version to version?
It was 1 in 5.6,
2 - in 5.7
3 - in 8.0

Reply:
Yes, innodb has more and more background threads.